rust/processor/src/schema.rs

```diff
@@ -518,6 +539,80 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    fungible_asset_activities (transaction_version, event_index) {
+        transaction_version -> Int8,
+        event_index -> Int8,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 66]
+        storage_id -> Varchar,
+        #[max_length = 1000]
+        asset_type -> Varchar,
+        is_frozen -> Nullable<Bool>,
+        amount -> Nullable<Numeric>,
+        #[sql_name = "type"]
+        type_ -> Varchar,
+        is_gas_fee -> Bool,
+        #[max_length = 66]
+        gas_fee_payer_address -> Nullable<Varchar>,
+        is_transaction_success -> Bool,
+        #[max_length = 1000]
+        entry_function_id_str -> Nullable<Varchar>,
+        block_height -> Int8,
+        #[max_length = 10]
+        token_standard -> Varchar,
```

Comment:
Is it better to use enums here? "v1" vs "v2" or "coin" vs "fungible_asset" seem like arbitrary strings that we don't have documentation for. cc @gregnazario as well

Reply:
I think it's fine to have `Varchar` so it's able to be extended if it's not supported on the schema level, as long as there is a matching enum.

Reply:
It's just v1 and v2 for now. We can always change this or do something else. We don't want enums in postgres btw it's hella complicated.

Reply:
yea open to suggestion on how these enums can be better. Maybe this should be handled in the graphql side cc @jjleng

Reply:
using Postgres enum is a double edge sword. Upgrading the enum would require DB schema upgrades, which is painful. I prefer to keep it as a string.

Reply:
How about Int4? And then we can declare a matching enum somewhere

Reply:
Interesting. Idk if it'll reflect in the API tho? We can explore a bit more here.